src/client/pages/Landing/Landing.tsx

```diff
@@ -203,7 +204,11 @@ export const Landing: React.FC = () => {
             {![Variation.IOS, Variation.ANDROID].includes(variation!) && (
               <>
                 <TopBar isTransparent>
-                  <LanguagePicker />
+                  {market === Market.Se ? (
+                    <PhoneNumber color="white" />
+                  ) : (
+                    <LanguagePicker />
+                  )}
```

Comment:
Are we sure that this should be implemented on landing page as well? How can a user switch language then?

Reply:
That makes sense. I wasn't sure before because I thought that everyone came to the landing page either through the website or the forever page and that the language was picked there but maybe they should be able to switch here too! :) I will remove it here either way, as people probably won't have any questions at this point anyway :P

Reply:
haha hopefully not - we can double-check with @zakkjellin as well :)